GameData/WildBlueIndustries/000WildBlueTools/Templates/ClassicStock/Storage/Hydrokarbon.txt

```diff
@@ -0,0 +1,354 @@
+RESOURCE_DEFINITION
+{
+	name = Hydrokarbon // LqdMethane
+	displayName = Hydrokarbon
+	abbreviation = HK
+	density = 0.00042561
+	unitCost = 0.05
+	hsp = 3348.8
+	vsp = 510000
+	flowMode = STACK_PRIORITY_SEARCH
+	transfer = PUMP
+	isTweakable = True
+	isVisible = True
+	color = 0.93, 0.87, 0.38
+	volume = 1
+}
+
+RESOURCE_DEFINITION
+{
+	name = RHK1 // Karbonite
```

Comment:
Can we get rid of references to Karbonite? I'd prefer to reduce the amount of finger pointing if people get upset